packages/components/src/components/inline-notification/inline-notification-styles.ts

```diff
@@ -14,8 +14,22 @@ import {
   getNotificationIconJssStyle,
   getNotificationRootJssStyle,
 } from './inline-notification-styles-shared';
+import { JssStyle } from 'jss';
+import { getTypographySlottedJssStyle } from '../../styles/typography-styles';
 
 const mediaQueryMaxS = getMediaQueryMax('s');
+const textStyles = (theme: Theme): JssStyle => ({
```

Comment:
I think we use the `get...JssStyle` convention when naming functions which return `JssStyle`. Same for `headingStyles` as well